src/dawn/Optimizer/PassInlining.h

```diff
@@ -34,7 +34,7 @@ namespace dawn {
 class PassInlining : public Pass {
 public:
   /// @brief Inlining strategies
-  enum InlineStrategyKind {
+  enum class InlineStrategyKind {
     IK_InlineProcedures,    ///< Inline functions with no return
```

Comment:
I'd remove `IK_` as it is obvious after the move to scoped enums.
My personal taste would also to rename `InlineStrategyKind` -> `InlineStrategy`.

Reply:
sure

Reply:
spawned this issue:
https://github.com/MeteoSwiss-APN/dawn/issues/241